designer/server/src/test/scala/pl/touk/nussknacker/ui/api/MigrationApiEndpointsSecuritySpec.scala

```diff
@@ -0,0 +1,108 @@
+package pl.touk.nussknacker.ui.api
+
+import io.circe.syntax._
+import io.restassured.RestAssured.`given`
+import io.restassured.module.scala.RestAssuredSupport.AddThenToResponse
+import org.scalatest.freespec.AnyFreeSpecLike
+import pl.touk.nussknacker.engine.api.process.ProcessName
+import pl.touk.nussknacker.engine.build.ScenarioBuilder
+import pl.touk.nussknacker.restmodel.validation.ValidationResults.ValidationResult
+import pl.touk.nussknacker.test.base.it.{NuItTest, WithRichConfigScenarioHelper}
+import pl.touk.nussknacker.test.config.WithRichDesignerConfig.TestCategory.Category1
+import pl.touk.nussknacker.test.config.{
+  WithMockableDeploymentManager,
+  WithRichConfigRestAssuredUsersExtensions,
+  WithRichDesignerConfig
+}
+import pl.touk.nussknacker.test.{NuRestAssureExtensions, NuRestAssureMatchers, RestAssuredVerboseLogging}
+import pl.touk.nussknacker.ui.process.marshall.CanonicalProcessConverter
+
+class MigrationApiEndpointsSecuritySpec
```

Comment:
we agreed that the naming convention for the tapir API specs looks like this:
```
{name of the http service}{Business|Security}Spec
``` 
could you please rename it?